libs/viewer/src/SimpleViewer.cpp

```diff
@@ -658,6 +659,56 @@ void SimpleViewer::updateUserInterface() {
         ImGui::Unindent();
     }
 
+    if (ImGui::CollapsingHeader("Camera")) {
+        ImGui::Indent();
+        //ImGui::SliderFloat("Focal length (mm)", &FilamentApp::get().getCameraFocalLength(), 16.0f, 90.0f);
```

Comment:
Why don't we keep the focal length and exposure parameters? They are super useful

Reply:
Doh!  Totally an accident.  Did this too fast.

Reply:
Fixed this, thanks for catching it.